### PR TITLE
Make recommendation to not CID for NAT stronger

### LINKED ISSUE #293: Stronger caution on using CID
@ianswett said in PR #272 on section "Passive Network Performance Measurement and Troubleshooting":

 "I think the caution against using Connection ID needs to be a lot stronger. Things will break, users will notice. I realize that implementations have to use this to enforce that, but I believe some already are. See a comment below as well."

### PR DESCRIPTION
**Pull request by mirjak**
fixes #293

### CODE REVIEW

File: draft-ietf-quic-manageability.md [hunk 1]
```diff
@@ -840,7 +840,7 @@ communicate with the client.
 The QUIC header optionally contains a connection ID which can be used as
 additional entropy beyond the 5-tuple, if needed. The QUIC handshake needs
 to be observed in order to understand whether the connection ID is present and
-what length it has. However, connection IDs may be renegotiated during
+what length it has. However, connection IDs may be renegotiated
```
Comment (mirjak) [inline draft-ietf-quic-manageability.md:843]:
```suggestion
what length it has. However, connection IDs may be renegotiated during
```

File: draft-ietf-quic-manageability.md [hunk 2]
```diff
@@ -849,9 +849,10 @@ discouraged for NAT applications.
 
 If a NAT hits an operational limit, it is recommended to rather drop the
```
Comment (ianswett) [inline draft-ietf-quic-manageability.md:850]:
```suggestion
If a NAT hits an operational limit, it is recommended to drop the
```

**Review by ianswett — APPROVED** [review]

File: draft-ietf-quic-manageability.md (hunk 2 above)
Comment (mirjak) [reply draft-ietf-quic-manageability.md:850]: This is addressed in another PR.

File: draft-ietf-quic-manageability.md [hunk 3]
```diff
@@ -849,9 +849,10 @@ discouraged for NAT applications.
...
 initial packets of a flow (see also {{sec-filtering}}),
-which potentially triggers a fallback to TCP, than using the connection ID to
-muliplex mulitple connections on the same IP address port pair which risks
-connectivity breakage later, in case the connection ID changes.
+which potentially triggers a fallback to TCP. Use of the connection ID to
+muliplex mulitple connections on the same IP address/port pair is not a
```
Comment (britram) [inline draft-ietf-quic-manageability.md:853]:
```suggestion
multiplex multiple connections on the same IP address/port pair is not a
```

**Review by britram — APPROVED** [review]

**Pull request merged**